common/src/main/java/org/apache/celeborn/common/network/server/ChunkStreamManager.java

```diff
@@ -41,14 +40,15 @@ public class ChunkStreamManager {
   private static final Logger logger = LoggerFactory.getLogger(ChunkStreamManager.class);
 
   private final AtomicLong nextStreamId;
+  // StreamId -> StreamState
   protected final ConcurrentHashMap<Long, StreamState> streams;
+  // ShuffleKey -> StreamId
+  protected final ConcurrentHashMap<String, Set<Long>> shuffleStreamIds;
```

Comment:
I think we need ConcurrentSet here

Reply:
It's already concurrentHashSet as value is ConcurrentHashMap.newKeyset